AwsEncryptionSDK/runtimes/go/examples/readme.md

```diff
@@ -0,0 +1,90 @@
+# AWS Encryption SDK for Go Examples
+
+This section features examples that show you
+how to use the AWS Encryption SDK.
+We demonstrate how to use the encryption and decryption APIs
+and how to set up some common configuration patterns.
+
+## APIs
+
+The AWS Encryption SDK provides two high-level APIs:
+one-step APIs that process the entire operation in memory
+and streaming APIs.
```

Comment:
I can't find the streaming API examples -- if it's just 1 or 2 examples could you link them specifically?
(I'm also surprised to hear this -- I was also under the impression that our Dafny libraries wouldn't support streaming until Dafny did some work. Did that get done?)

Reply:
No. This came in when I copied the readme from other runtimes. I think we don't support streaming rn.